src/2025h2/FLS-up-to-date-capabilities.md

```diff
@@ -0,0 +1,64 @@
+# Develop the capabilities to keep the FLS up to date
+
+| Metadata         |                                                                                  |
+|:-----------------|----------------------------------------------------------------------------------|
+| Point of contact | *@JoelMarcey*                                                                    |
+| Teams            | &lt;!-- #t-spec, #t-lang, #t-opsem --&gt;                                                  |
+| Task owners      | &lt;!-- @JoelMarcey --&gt;                                                       |
+| Status           | Proposed, Invited                                                                |
```

Comment:
Since you'll be doing a bunch of work, rather than just proposing the goal:

```suggestion
| Status           | Proposed                                                                         |
```